pkg/provider/interface.go

```diff
@@ -0,0 +1,7 @@
+package provider
```

Comment:
What about a file called labeller.go with both in?

Reply:
Done